### DISCUSSION TITLE
Is Pinia available for none Vue applications

### DISCUSSION THREAD

**Discussion by MrAnyx** [vuejs/pinia#1720] [Help and Questions] [answered]
I'm wondering if there is a way to configure and use Pinia with a none Vue application. I really like how Pinia works and I would like to use it in my personal projet but it doesn't use Vue.

Is it currently possible ?

**Comment by posva** [comment, marked as answer]
As long as you use the [uve reactivity api](https://vuejs.org/api/reactivity-core.html#watch), pinia will work with watchers, computed, etc
